hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/utils/EnrichedSpanUtils.java

```diff
@@ -124,6 +162,10 @@ public static Protocol getProtocol(Event event) {
     return null;
   }
 
+  private static boolean nullCheck(AttributeValue attributeValue) {
+    return attributeValue != null && attributeValue.getValue() != null;
```

Comment:
Tip: An alternative way for chained null checks would be
`Optional.ofNullable(attributeValue).map(AttributeValue::getValue).filter(Objects::nonNull).isPresent()`.

This allows us later to introduce null check one more level down by introducing something like `.map(Value::getString)` for instance.